docs/query-builder.txt

```diff
@@ -381,6 +381,41 @@ wildcard characters:
          ...
       ]
 
+whereLike() and whereNotLike() Methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following methods provide the same functionality as using the
+``like`` query operator to match patterns:
```

Comment:
Even though it's right above this, it might be good to link to the `like` operator here